packages/core/test/acceptance/i18n_spec.ts

```diff
@@ -0,0 +1,964 @@
+    it('inside HTML elements', () => {
+      ɵi18nConfigureLocalize({
+        translations: {
+          '{VAR_PLURAL, plural, =0 {no {$startBoldText}emails{$closeBoldText}!} =1 {one {$startItalicText}email{$closeItalicText}} other {{$interpolation} {$startTagSpan}emails{$closeTagSpan}}}':
+              '{VAR_PLURAL, plural, =0 {aucun {$startBoldText}email{$closeBoldText}!} =1 {un {$startItalicText}email{$closeItalicText}} other {{$interpolation} {$startTagSpan}emails{$closeTagSpan}}}',
+          '{VAR_SELECT, select, other {(name)}}': '{VAR_SELECT, select, other {({$interpolation})}}'
+        }
+      });
+      const fixture = initWithTemplate(AppComp, `<div i18n><span>{count, plural,
+        =0 {no <b>emails</b>!}
+        =1 {one <i>email</i>}
+        other {{{count}} <span title="{{name}}">emails</span>}
+      }</span> - <span>{name, select,
+        other {({{name}})}
+      }</span></div>`);
+      expect(fixture.nativeElement.innerHTML)
+          .toEqual(
+              `<div><span>aucun <b>email</b>!<!--ICU 9--></span> - <span>(Angular)<!--ICU 15--></span></div>`);
+
+      fixture.componentRef.instance.count = 4;
+      fixture.detectChanges();
+      expect(fixture.nativeElement.innerHTML)
+          .toEqual(
+              `<div><span>4 <span title="Angular">emails</span><!--ICU 9--></span> - <span>(Angular)<!--ICU 15--></span></div>`);
+
+      fixture.componentRef.instance.count = 0;
+      fixture.componentRef.instance.name = 'John';
+      fixture.detectChanges();
+      expect(fixture.nativeElement.innerHTML)
+          .toEqual(
+              `<div><span>aucun <b>email</b>!<!--ICU 9--></span> - <span>(John)<!--ICU 15--></span></div>`);
+    });
+
+    it('inside template directives', () => {
+      ɵi18nConfigureLocalize({
+        translations: {
+          '{VAR_SELECT, select, other {(name)}}': '{VAR_SELECT, select, other {({$interpolation})}}'
+        }
+      });
+      const fixture = initWithTemplate(AppComp, `<div i18n><span *ngIf="visible">{name, select,
+        other {({{name}})}
+      }</span></div>`);
+      expect(fixture.nativeElement.innerHTML)
+          .toEqual(`<div><span>(Angular)<!--ICU 4--></span><!--bindings={
+  "ng-reflect-ng-if": "true"
+}--></div>`);
+
+      fixture.componentRef.instance.visible = false;
+      fixture.detectChanges();
+      expect(fixture.nativeElement.innerHTML).toEqual(`<div><!--bindings={
+  "ng-reflect-ng-if": "false"
+}--></div>`);
+    });
+
+    it('inside ng-container', () => {
+      ɵi18nConfigureLocalize({
+        translations: {
+          '{VAR_SELECT, select, other {(name)}}': '{VAR_SELECT, select, other {({$interpolation})}}'
+        }
+      });
+      const fixture = initWithTemplate(AppComp, `<ng-container i18n>{name, select,
+        other {({{name}})}
+      }</ng-container>`);
+      expect(fixture.nativeElement.innerHTML).toEqual(`(Angular)<!--ICU 4--><!--ng-container-->`);
+    });
+
+    it('should handle ICUs inside <ng-template>', () => {
```

Comment:
Nit: for readability

```suggestion
    it('inside <ng-template>', () => {
```

Otherwise the test reads "should support ICU expressions should handle ICUs inside ng-template"

Reply:
This still needs updating